### PR TITLE
Add time to log output

### PR DESCRIPTION
**Pull request by timneutkens**
screen shot 2017-02-15 at 22 15 12

### CODE REVIEW

**Comment by timneutkens** [comment]
This actually breaks all tests since they depend on output 😅

**Comment by timneutkens** [comment]
@geowarin in what way would you want to fix the test?

**Comment by geowarin** [comment]
Woah it looks great :smile: 

I guess the best approach here is not to capture the timestamp in tests.
It might need a bit of refactoring.

**Comment by geowarin** [comment]
Something like

```js
  logOnly () {
    if (this.enabled) {
      console.log(arguments) // pseudo-code :)
    }
  }
```

**Comment by timneutkens** [comment]
@geowarin I'm thinking we could also just check for `process.env.NODE_ENV !== 'test'` right?

**Comment by geowarin** [comment]
Sure, good idea!

**Comment by timneutkens** [comment]
Fixed 🙌

File: src/output.js [hunk 1]
```diff
@@ -48,9 +49,26 @@ class Debugger {
...
+        this.log(message);
+        this.log();
+        return;
+      }
+
+      // Make timestamp appear at the end of the line
+      let logSpace = process.stdout.columns - stringLength(message) - stringLength(dateString)
+      if (logSpace <= 0) {
+        logSpace = 10
+      }
```
Comment (geowarin) [inline src/output.js]: I'm a bit confused... What happens if we don't have enough space to display the date ?

Comment (timneutkens) [inline src/output.js]: This makes sure there's always a little bit of space between the title message and the date.

screen shot 2017-02-16 at 01 00 59

Comment (geowarin) [inline src/output.js]: :+1:

**Comment by geowarin** [comment]
LGTM, just needs rebasing and I'll ship 1.3.0 release

**Comment by timneutkens** [comment]
Was already working on that 😄

**Comment by timneutkens** [comment]
😍

**Pull request merged**